pkgs/servers/mail/archiveopteryx/default.nix

```diff
@@ -0,0 +1,16 @@
+{ stdenv, fetchurl, openssl, perl, zlib }:
+stdenv.mkDerivation rec {
+  version = "3.2.0";
+  name = "archiveopteryx-${version}";
+  src = fetchurl {
+    url = "http://archiveopteryx.org/download/${name}.tar.bz2";
+    sha256 = "0i0zg8di8nbh96qnyyr156ikwcsq1w9b2291bazm5whb351flmqx";
+  };
+  buildInputs = [ openssl perl zlib ];
+  installPhase = ''
+    INSTALLROOT=$out make install
+    mkdir $out/bin
+    ln --symbolic $out/usr/local/archiveopteryx/sbin/* $out/bin/
+    ln --symbolic $out/usr/local/archiveopteryx/bin/* $out/bin/
+  '';
```

Comment:
I was a bit lost determining which BSD license this is https://github.com/aox/aox/blob/master/doc/readme/license.txt
But then I found https://github.com/aox/aox/blob/master/doc/readme/COPYING. PostgreSQL License it is.